lib/iris/fileformats/grib/_message.py

```diff
@@ -376,7 +376,7 @@ def _get_key_value(self, key):
                        'satelliteNumber', 'instrumentType',
                        'scaleFactorOfCentralWaveNumber',
                        'scaledValueOfCentralWaveNumber',
-                       'longitudes', 'latitudes', 'distinctLatitudes')
+                       'longitudes', 'latitudes')
```

Comment:
OK, so having these listed in the _Section class is safe, because they can only be accessed as coded keys if gribapi knows this to be true for the particular message being handled.